Jenkinsfile

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env groovy
+
+REPOSITORY = 'rummager'
+
+node {
+  def govuk = load '/var/lib/jenkins/groovy_scripts/govuk_jenkinslib.groovy'
+
+  properties([
+    buildDiscarder(
+      logRotator(
+        numToKeepStr: '50')
+      ),
+    [$class: 'RebuildSettings', autoRebuild: false, rebuildDisabled: false],
+    [$class: 'ThrottleJobProperty',
+      categories: [],
+      limitOneJobWithMatchingParams: true,
+      maxConcurrentPerNode: 1,
+      maxConcurrentTotal: 0,
+      paramsToUseForLimit: 'Rummager',
+      throttleEnabled: true,
+      throttleOption: 'category'],
+  ])
+
+  try {
+    stage("Checkout") {
+      checkout scm
+    }
+
+    stage("git merge") {
+      govuk.mergeMasterBranch()
+    }
+
+    stage("bundle install") {
+      govuk.bundleApp()
+    }
+
+    stage("Delete static symlinks and reconnect") {
+      sh '''for d in images javascripts templates stylesheets; do
+              rm -f public/$d
+              ln -s ../../../Static/workspace/public/$d public/
+            done
+         '''
+    }
+
+    stage("rubylinter") {
+      govuk.rubyLinter()
```

Comment:
@MatMoore do you know if there's any difference between running https://github.com/alphagov/rummager/blob/master/jenkins.sh#L20-L25 and https://github.com/alphagov/govuk-puppet/blob/master/modules/govuk_jenkins/files/var/lib/jenkins/groovy_scripts/govuk_jenkinslib.groovy#L54-L59 in this repo?